target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -111,6 +111,17 @@ define Device/tplink_archer-c6-v2-us
 endef
 TARGET_DEVICES += tplink_archer-c6-v2-us
 
+define Device/tplink_tl-wa1201-v2
```

Comment:
Sorting. And I really do wonder why you chose this particular place for the entry in the first place.

Reply:
I guess its kinda random. Do I put this at the bottom of the tp-link devices or is there some other logic?